packages/Main/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "itowns",
+  "version": "2.44.2",
+  "description": "A JS/WebGL framework for 3D geospatial data visualization",
+  "type": "module",
+  "main": "lib/Main.js",
+  "exports": {
+    ".": "./lib/Main.js"
+  },
+  "scripts": {
+    "lint": "eslint \"src/**/*.{js,ts,tsx}\" \"test/**/*.js\"",
+    "transpile": "cross-env BABEL_DISABLE_CACHE=1 babel src --out-dir lib",
+    "test-unit": "cross-env BABEL_ENV=test npm run base-test-unit test/unit",
+    "base-test-unit": "cross-env BABEL_DISABLE_CACHE=1 mocha --timeout 5000 --file test/unit/bootstrap.js --import=../../config/babel-register/register.mjs",
+    "test-with-coverage": "c8 -n src -r html cross-env npm run test-unit",
+    "test-with-coverage_lcov": "c8 -n src --reporter=lcov cross-env npm run test-unit",
+    "watch": "cross-env BABEL_DISABLE_CACHE=1 babel --watch src --out-dir lib",
+    "postinstall": "cross-env NO_UPDATE_NOTIFIER=true node ./scripts/prepare.mjs && node ./scripts/replace.config.mjs",
+    "prepublishOnly": "npx copyfiles -u 1 \"../../examples/**/*\" ./examples/ && npx copyfiles -u 1 \"../../docs/**/*\" ./docs/ && npx copyfiles -u 1 \"../../dist/**/*\" ./dist/ ",
```

Comment:
Or just not distribute examples and docs like most other packages.

Reply:
to be decided